src/main/resources/standardTestCases/invalidate_bad_caches_jes_no_copy.test

```diff
@@ -0,0 +1,15 @@
+name: invalidate_bad_caches_jes_no_copy
+testFormat: workflowsuccess
+backends: [Jes-Caching-No-Copy]
+
+files {
+  wdl: invalidate_bad_caches/invalidate_bad_caches_no_copy.wdl
+}
+
+metadata {
+    "calls.invalidate_bad_caches.make_file.callCaching.result": "Cache Miss"
+    "calls.invalidate_bad_caches.make_file.callCaching.allowResultReuse": false
+    "calls.invalidate_bad_caches.invalidate_cache_and_remake_file.callCaching.allowResultReuse": true
+    "calls.invalidate_bad_caches.delete_file_in_gcs.callCaching.result": "Cache Miss"
```

Comment:
Hmmm I'm not entirely sure why this test case can't be used for the backends "Jes" and "Jes-Caching-No-Copy" with backendMode "all"?